backend/src/docs/v1.api-spec.yaml

```diff
@@ -314,7 +333,7 @@ components:
           example: MIGfMA0GCSqGS...
         clientEnvironment:
           type: string
-          description: Which WebADE environment to work on - one of INT, TEST or PROD
+          description: 'Which WebADE environment to work on - one of INT, TEST or PROD'
```

Comment:
You can add a string enumeration here to specify programatically what environment strings are allowed/expected in addition to the description comment.

Reply:
Good idea. Have added the enum values now in a number of spots